superset/advanced_data_type/api.py

```diff
@@ -96,15 +90,9 @@ def get(self, **kwargs: Any) -> Response:
             500:
               $ref: '#/components/responses/500'
         """
-        items = kwargs["rison"]
-        advanced_data_type = items.get("type")
-        if not advanced_data_type:
```

Comment:
Rison automatically parses the query string from the defined json schema

Reply:
Good catch 👍